components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/ApplicationManagementService.java

```diff
@@ -331,6 +332,16 @@ public abstract boolean isExistingApplicationTemplate(String templateName, Strin
     public abstract List<SpTemplate> getAllApplicationTemplateInfo(String tenantDomain)
             throws IdentityApplicationManagementException;
 
+    /**
+     * Get inbound authenticator configurations
```

Comment:
It would be better if we could give an idea of the key and values of the map. For example, what is the key returned in this map?